app/views/tasks/index.html.slim

```diff
@@ -42,12 +43,15 @@
           p
             b = t('tasks.index.search.order')
           .btn-group[role="group"]
-            = sort_link(@search, :created_at)
-        /.stars.section
+            div
+              = sort_link(@search, :created_at)
+            div
+              = sort_link(@search, :average_rating)
```

Comment:
At least in my tests performed locally, this sorting is not working as expected. Would you mind having a look here again, please?